src/ordering/cuthillMckeeOrdering_driver.cpp

```diff
@@ -75,7 +87,7 @@ void do_cuthillMckeeOrdering(
 
         pgrouting::UndirectedGraph undigraph(gType);
         undigraph.insert_edges(data_edges, total_edges);
-
+        // auto results = cuthillMckeeOrdering(undigraph, start_vid);  // might cause error bcz of type
```

Comment:
Is this how you tell the compiler to not use this code?

Reply:
Will keep this in mind